README.md

```diff
@@ -34,60 +34,53 @@
-
+Glasskube is an **Open Source package manager for Kubernetes**.
+It makes deploying, updating, and configuring packages on Kubernetes **20 times faster** than tools like **Helm or Kustomize**.
+Inspired by the simplicity of Homebrew and npm. You can decide if you want to use the Glasskube UI, CLI, or directly deploy packages via GitOps.
 
 ## ⭐️ Why Glasskube?
 
-Using **traditional package managers** or applying manifests directly can be **super confusing** and doesn't scale.
-Therefore, Glasskube will help you to **install your favorite Kubernetes packages** using the **Glasskube UI** for reduced complexity and increased transparency.
-We are also providing a **brew inspired CLI** for advanced users. Our **packages are dependency aware**, as you would expect from a package manager.
-Designed as a cloud native application, so you can follow your **GitOps approach**.
-
-## ✨ Features
+We have been working in the Kubernetes ecosystem for over five years.
+During this time, we have consistently struggled with package management, configuration, and distribution.
+We've spent countless hours templating and writing documentation for commands and concepts that were difficult to grasp.
 
-- 💡 **Streamlined UI and CLI Experience**:
-  <br> We've stripped away unnecessary complexities, providing a simple yet powerful user interface and command-line interface for easy package management.
-- 🔄 **Automated Updates**:
-  <br> Glasskube ensures your Kubernetes packages and apps are always up-to-date, minimizing the manual effort required for maintenance.
-- 🤝 **Dependency Awareness**:
-  <br> We understand the interconnected nature of Kubernetes packages. Glasskube intelligently manages dependencies.
-- 🛠️ **GitOps Ready** with ArgoCD or Flux:
-  <br> Seamlessly integrate Glasskube into your GitOps workflow with support for popular tools like ArgoCD or Flux.
-- 📦 **Central Package Repository**:
-  <br> Keep track of all your packages in one central repository, with a [planned](https://glasskube.dev/roadmap/?utm_source=github) feature for custom repositories.
-- 🔍 **Cluster Scan** ([planned](https://glasskube.dev/roadmap/?utm_source=github)):
-  <br> Introducing the Cluster Scan feature in a future version, which allows you to detect packages in your cluster, providing valuable insights for better management and upgrade paths.
-- 🔐 **Version Pinning**:
-  <br> With a future version, Glasskube will introduce Version Pinning, giving you control over your package versions.
+In contrast, tools like Homebrew, apt, and dnf felt easy to use and rarely caused problems.
+While working on other cloud-native projects, users consistently highlighted several common pain points.
```

Comment:
I think this sentence could be expressed more clearly. Does it mean:
1. "_We_ worked on other projects and _our_ users had problems", or
2. "Users working on _their_ projects had problems"?

I think the first interpretation makes more sense from the context but the sentence reads more like the second, so I was confused.